### PR TITLE
add merge blocking to ci: sev template

### PR DESCRIPTION
**Pull request by clee2000**
as in title, so that by default, ci: sev will block merges

the line can be removed to not block merges

### CODE REVIEW

**Review by huydhn — APPROVED** [review]
Ohhh, this is how it works.  Do this block merge completely or like -f still work? I guess it's the former.

**Comment by janeyx99** [comment]
-f would still work

File: .github/ISSUE_TEMPLATE/ci-sev.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ about: Tracking incidents for PyTorch's CI infra.
 
 > NOTE: Remember to label this issue with "`ci: sev`"
 
+**MERGE BLOCKING** <!-- remove this if you don't want to block merges -->
```
Comment (janeyx99) [inline .github/ISSUE_TEMPLATE/ci-sev.md:8]:
```suggestion
**MERGE BLOCKING** <!-- remove this tag if you don't want this SEV to block merges -->
```

Comment (janeyx99) [reply .github/ISSUE_TEMPLATE/ci-sev.md:8]: ok the tag part is extraneous, but I think we can be slightly clearer :p

**Review by janeyx99 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**